flang/test/Driver/emit-fir.f90

```diff
@@ -1,4 +1,4 @@
-! RUN: %flang_fc1 -emit-fir %s -o - | FileCheck %s
+! RUN: %flang_fc1 -emit-mlir %s -o - | FileCheck %s
```

Comment:
Nit: Do you want to rename the file also?

Reply:
Thanks for pointing this out - updated!